tests/installation/partitioning_iscsi.pm

```diff
@@ -13,14 +13,21 @@ use testapi;
 
 sub run() {
     send_key "alt-c";    # create partition setup
-    sleep 2;
+    wait_still_screen(2);
     assert_screen "preparing-disk-select-iscsi-disk";
     send_key "alt-1";    # select ISCSI disk
     send_key "alt-n";    # next
-    assert_screen "preparing-disk-install-on";
-    send_key "alt-e";    # use entire iscsi disk
-    send_key "alt-n";    # next
-    assert_screen "edit-proposal-settings";
+    if (check_screen("preparing-disk-use-entire-disk-button", 10)) {
+        send_key "alt-e";    # use entire iscsi disk
+        send_key "alt-d";    # edit proposal settings
+    }
+    else {
+        send_key "alt-e";    # edit proposal settings
+    }
+    assert_screen "preparing-disk-proposal-settings";
```

Comment:
would you mind combining this assert_screen with the check_screen in line 20 as in
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f9aaf1b4c8f7f2e977f8d1b5b0cbbc726bd89255/tests/x11/sshxterm.pm#L19 and following?